release/release_gold.sh

```diff
@@ -89,12 +89,12 @@ function commit_to_branch_and_send_pull_request {
 }
 
 function get_tag_name {
-	if [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "dxp" ]
-	then
-		echo "${_ARTIFACT_VERSION}"
-	elif [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "portal" ]
+	if (is_ga_release || is_u_release)
```

Comment:
@davisaints, can you please include a test for checking the releases `7.4 U`?

Thanks